README.md

```diff
@@ -34,10 +34,15 @@ We aim to support a range of pose estimation packages, along with 2D or 3D track
 
 Find out more on our [mission and scope](https://movement.neuroinformatics.dev/community/mission-scope.html) statement and our [roadmap](https://movement.neuroinformatics.dev/community/roadmaps.html).
 
-## Status
 > [!Warning]
 > 🏗️ The package is currently in early development and the interface is subject to change. Feel free to play around and provide feedback.
 
+> [!Tip]
+> If you prefer analysing your data in R, we recommend checking out the
+> [animovement](https://www.roald-arboel.com/animovement/) toolbox, which is similar in scope.
+> We are working together with its developer
+> to gradually converge on common data standards and workflows.
```

Comment:
Just fyi, for the script I pasted to work as is the admonition needs to be in one line
```suggestion
> If you prefer analysing your data in R, we recommend checking out the [animovement](https://www.roald-arboel.com/animovement/) toolbox, which is similar in scope. We are working together with its developer to gradually converge on common data standards and workflows.
```

Reply:
I modified the script such that it can correctly parse both cases, so the suggested modification is no longer necessary.